scripts/mempool.js

```diff
@@ -250,10 +257,10 @@ export class Mempool {
      * Get the total wallet balance
      * @param {UTXO_WALLET_STATE} filter the filter you want to apply
      */
-    getBalance(filter, includeLocked = false) {
+    getBalance(filter, includeLocked = false, includeImmature = false) {
```

Comment:
I don't like all these parameters, can you include these in the filter?

Reply:
if I added them filter params would grow exponentially
SPENDABLE_LOCKED_IMMATURE
SPENDABLE_LOCKED_MATURE
SPENDABLE_UNLOCKED_IMMATURE
SPENDABLE_LOCKED_IMMATURE
.... and so on for cold staking filters...

I think that 3 params in getBalance are ok

Reply:
ok I think I understood what you meant, review applied